JSDemos/Demos/Resizable/Overview/description.md

```diff
@@ -1 +1,21 @@
-The Resizable is a UI component that enables its content to be resizable in the UI.
\ No newline at end of file
+The Resizable enables you to resize its content. In this demo you can drag [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) to resize the DataGrid.
+
+Configure the following properties to specify the boundaries to which you can drag the handles:
+
+- [minWidth](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#minWidth)    
+The lower width boundary.
+
+- [maxWidth](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#maxWidth)    
+The upper width boundary.
+
+- [minHeight](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#minHeight)    
+The lower height boundary.
+
+- [maxHeight](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#maxHeight)    
+The upper height boundary.
+
+You can specify four types of handles: *top*, *bottom*, *left* and *right*. Separate handles by a space in the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) property to specify which handles to display in the UI. Combine nearest handles (for example, "bottom left") to show corner handles.
```

Comment:
A few notes: 

- You say "four types". But you also say that if I combine values ("bottom right"), I can display corner handles. To that amounts to eight types of handles total (four on edges and four in corners). Obviously you can't display them all at the same time, but those are the possible types, it seems. 
- I wouldn't explain the separator character here in the demo description, for two reasons. One - they can look it up if they need. Two - you give an example that uses a space, so that already gives readers a hint. 
- "bottom left" is not the best example. The most commonly used handle is "bottom right". I can see it right now in this GitHub editor. So I would use that as an example. 

All in all, these comments made me come up with the following suggestion:

```suggestion
You can display resize handles on edges or corners. Use the following keywords to set up the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) property: *top*, *bottom*, *left* and *right*. If you specify two adjacent sides (for example, "bottom right"), the control displays a single handle in the corner.
```